app/uk/gov/hmrc/hec/models/CorrectiveAction.scala

```diff
@@ -25,8 +25,8 @@ import play.api.libs.json.Format
 sealed trait CorrectiveAction extends EnumEntry with Product with Serializable
 
 object CorrectiveAction extends Enum[CorrectiveAction] {
-  case object Register extends CorrectiveAction
-  case object Dormant extends CorrectiveAction
+  case object RegisterNewSaAccount extends CorrectiveAction
```

Comment:
do we need one for companies too?

Reply:
I think we will be using the same for company

Reply:
here I thought we could get away with having the one thing so that we don't need to map two things for the CDW enum file. It does open up the possibility of `RegisterNewSaAccount` appearing for companies though which isn't ideal. So in theory it probably is better to keep it separate like we do for the FE ones

Reply:
although, we only send one enum file down to CDW and we're currently using enumeratum to collect all the different corrective action options, so having it separate for individuals and companies messes that up a little. If we had them separate we'd need to collect them all again for the enum file and somehow make sure we don't duplicate any enum values

Reply:
can we just call it `RegisterNewAccount `?

Reply:
the description for it in the enum file is "Register new SA account" - we'd have a separate one for companies I think if we ever had "register new account" for companies in the future

Reply:
oh ok, so currently we don't do corrective actions for companies?

Reply:
We do, but not the new one, it's only reactivated / other

Reply:
ah ok, finally understood 👍 thanks!